### PR TITLE
Add files via upload: Upload logo CFB and tokens information

### PR DESCRIPTION
**Pull request by cryptocfb**

### CODE REVIEW

**Comment by bobcoin98** [comment]
Please change filenames to lowercase format
(This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) 
 On behalf of [bobcoin98](https://github.com/bobcoin98)

**Comment by cryptocfb** [comment]
> Please change filenames to lowercase format (This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) On behalf of [bobcoin98](https://github.com/bobcoin98)
filename is lowercase format, why need change?
What filenames of ? Logo or token name? of something else? i do not understand what you mean?

**Review by bobcoin98 — CHANGES_REQUESTED** [review]
Please follow  this 
https://github.com/BuildOnViction/tokens#2-create-a-pull-request

**Comment by cryptocfb** [comment]
> Please follow this https://github.com/BuildOnViction/tokens#2-create-a-pull-request

Yes, we have update logo filename to lowercase here : https://drive.google.com/file/d/1buOD_Op_WPbtPEEe9qV8UBRS7og4XRyG/view?usp=sharing

**Comment by bobcoin98** [comment]
image

**Comment by bobcoin98** [comment]
`B93` ????

**Review by bobcoin98 — CHANGES_REQUESTED** [review]
please update .png filename as weel

**Comment by cryptocfb** [comment]
> please update .png filename as well

Where to update again? can not edit file png again? where to publish again?
Our error because of your copy on https://tomoscan.io/token/0x567b1e373f7285e9a32f5eeb9305b6964b6a3532
when we copy smc it auto change to B from b, please fix it 
But now, how to rename .png file?
i can not edit that png file

**Comment by bobcoin98** [comment]
update filename same you did with json file

**Comment by cryptocfb** [comment]
[0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json](https://github.com/BuildOnViction/tokens/files/13327815/0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json)
0x567b1e373f7285e9a32f5eeb9305b6964b6a3532

**Comment by cryptocfb** [comment]
https://drive.google.com/file/d/13Y_Foyz-n84w9aV65LpzhqxZTz3_ZWsQ/view?usp=sharing
https://drive.google.com/file/d/1QtaM0C-xsvepBuZejNT0nhwr9jJlXTGP/view?usp=sharing

**Review by cryptocfb — COMMENTED** [review]
0x567b1e373f7285e9a32f5eeb9305b6964b6a3532

**Review by cryptocfb — COMMENTED** [review]
0x567b1e373f7285e9a32f5eeb9305b6964b6a3532

**Review by cryptocfb — COMMENTED** [review]
[0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json](https://github.com/BuildOnViction/tokens/files/13327853/0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json)

**Comment by cryptocfb** [comment]
https://github.com/tomochain/tokens/issues/401

**Comment by cryptocfb** [comment]
File has been updated, please promptly upload the CFB logo to the Tomo
blockchain network. (
https://github.com/BuildOnViction/tokens/pull/400/commits/271ccf01891624b25c338af508d57baa35b2c5f4).
Thank you.

Vào CN, 12 thg 11, 2023 vào lúc 16:37 Bob Jr ***@***.***>
đã viết:

**Comment by cryptocfb** [comment]
> update filename same you did with json file

Done, pls update logo for us on Tomo(Vic) Wallet, C98 Wallet and Tomoscan.
Thanks

**Comment by bobcoin98** [comment]
Rule violation
 - Number of files: only 2 file changes accepted 
 Logo size: 256x256  ( 512x512 detected)
(This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) 
 On behalf of [bobcoin98](https://github.com/bobcoin98)

**Review by bobcoin98 — CHANGES_REQUESTED** [review]
follow steps here : https://github.com/BuildOnViction/tokens#2-create-a-pull-request

[2. Create a pull request](https://github.com/BuildOnViction/tokens/pull/400/files#2-create-a-pull-request)

[Image Requirements](https://github.com/BuildOnViction/tokens/pull/400/files#image-requirements)

file extension: png (PNG is invalid)
name：contract_address.png (filename in lowercase). Example: 0xea8d13af16a33fc5de19c0b1edcf4cd053bd2fa9.png
Resolution: recommend 256x256 px
size: under 512 KB
background: preferably transparent
(use simple drag and drop online service [tinypng](https://tinypng.com/) to optimize image size)

[Token Information Requirement](https://github.com/BuildOnViction/tokens/pull/400/files#token-information-requirement)

file extension: json (JSON is invalid)
name：contract_address.json (filename in lowercase). Example: 0xea8d13af16a33fc5de19c0b1edcf4cd053bd2fa9.json
required fields: symbol, name, website

**Comment by cryptocfb** [comment]
> follow steps here : https://github.com/BuildOnViction/tokens#2-create-a-pull-request
> 
> [2. Create a pull request](https://github.com/BuildOnViction/tokens/pull/400/files#2-create-a-pull-request)
> 
> [Image Requirements](https://github.com/BuildOnViction/tokens/pull/400/files#image-requirements)
> 
> file extension: png (PNG is invalid) name：contract_address.png (filename in lowercase). Example: 0xea8d13af16a33fc5de19c0b1edcf4cd053bd2fa9.png Resolution: recommend 256x256 px size: under 512 KB background: preferably transparent (use simple drag and drop online service [tinypng](https://tinypng.com/) to optimize image size)
> 
> [Token Information Requirement](https://github.com/BuildOnViction/tokens/pull/400/files#token-information-requirement)
> 
> file extension: json (JSON is invalid) name：contract_address.json (filename in lowercase). Example: 0xea8d13af16a33fc5de19c0b1edcf4cd053bd2fa9.json required fields: symbol, name, website

I want delete all and upload all files again? how to do that? I can not delete pgn file, show me, thanks

**Comment by cryptocfb** [comment]
I want delete all and upload all files again? how to do that? I can not delete pgn file, show me, thanks, it do not show delete files
cfb

**Comment by cryptocfb** [comment]
> follow steps here : https://github.com/BuildOnViction/tokens#2-create-a-pull-request
> 
> [2. Create a pull request](https://github.com/BuildOnViction/tokens/pull/400/files#2-create-a-pull-request)
> 
> [Image Requirements](https://github.com/BuildOnViction/tokens/pull/400/files#image-requirements)
> 
> file extension: png (PNG is invalid) name：contract_address.png (filename in lowercase). Example: 0xea8d13af16a33fc5de19c0b1edcf4cd053bd2fa9.png Resolution: recommend 256x256 px size: under 512 KB background: preferably transparent (use simple drag and drop online service [tinypng](https://tinypng.com/) to optimize image size)
> 
> [Token Information Requirement](https://github.com/BuildOnViction/tokens/pull/400/files#token-information-requirement)
> 
> file extension: json (JSON is invalid) name：contract_address.json (filename in lowercase). Example: 0xea8d13af16a33fc5de19c0b1edcf4cd053bd2fa9.json required fields: symbol, name, website

We have uploaed 2 files follow your guide at here https://github.com/BuildOnViction/tokens/pull/400/files
Please check it, is it ok?
Thanks

File: 0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "symbol": "CFB",
+    "name": "Crypto For Betting",
+    "description": "Cryptocurrency is utilized for betting on global lottery results and wagering in online roulette games",
+    "website": "https://lottocfb.com",
+    "communities": {
+        "email": "cryptocfb@gmail.com",
```
Comment (bobcoin98) [inline 0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json:7]: please use official project email
in your case, use mail ....@lottocfb.com

**Review by bobcoin98 — CHANGES_REQUESTED** [review]

File: 0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json (hunk 1 above)
Comment (cryptocfb) [reply 0x567b1e373f7285e9a32f5eeb9305b6964b6a3532.json:7]: > please use official project email personal email is not accepted in your case, use mail [....@lottocfb.com](mailto:....@lottocfb.com)

Fuck you guys, you guys are so troublesome and a waste of users' time
1. In the instructions, is there any place where you must write that you must use an official email ending with lottocfb.com to be allowed? I don't see anywhere that says it's required to have an email extension like that to be allowed. Why can't I use gmail? Can you show me what conditions require that? It's not in the instructions, why ask now?
2. Support time is too long, it takes weeks for someone to respond after posting. We have created many tokens on the Tomochain platform and have never been required to use an official email to update. You can use gmail
3. You have few users and have a list of requirements that meet all the conditions, most of which are due to your errors (for example, the error of copying the smart contract on tomoscan automatically turns the letter b into a capital letter B, which also makes users make a mistake, even though it's your fault)
4. Please update the logo for us after we meet your basic requirements, and do not have any additional requests beyond the initial requirements.
5. Fuck you guys, fuck C98, fuck Tomochain for doing business like a dick

**Comment by cryptocfb** [comment]
We have updated email official, please confirm is it ok? 
Update our logo soon for C98 wallet and Tomoscan, do not waste our time
Thanks

**Review by bobcoin98 — COMMENTED** [review]
This PR seems to be OK. Merging it now !
(This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) 
 On behalf of [bobcoin98](https://github.com/bobcoin98)

**Comment by bobcoin98** [comment]
oops, wrong folder 
revert

**Comment by cryptocfb** [comment]
> oops, wrong folder revert

is it done?

**Comment by bobcoin98** [comment]
done in my other commit
check here 
https://tomoscan.io/token/0x567b1e373f7285e9a32f5eeB9305b6964B6a3532

**Comment by cryptocfb** [comment]
> done in my other commit
> check here
> https://tomoscan.io/token/0x567b1e373f7285e9a32f5eeB9305b6964B6a3532

But social link telegram, website, twitter have not update, why you ask us fill social link but you not update on tomoscan? Please update it

**Comment by cryptocfb** [comment]
> done in my other commit
> check here
> https://tomoscan.io/token/0x567b1e373f7285e9a32f5eeB9305b6964B6a3532

Please update link telegram, twitter, .. on https://scan.tomochain.com/tokens/0x567b1e373f7285e9a32f5eeb9305b6964b6a3532

**Comment by cryptocfb** [comment]
Why do you force us to update telegram social network links, emails (with official domain), and all other strict requirements, but after we follow those requirements, you don't update those links to Tomoscan? Why is it so funny? Are you kidding us? So why ask us to update? Why and WHY AND WHY?

**Pull request merged**